agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -503,6 +508,28 @@ class backhaul_manager : public btl::transport_socket_thread {
                           const sLinkNeighbor &link_neighbor, const sLinkMetrics &link_metrics,
                           ieee1905_1::eLinkMetricsType link_metrics_type);
 
+    typedef struct sApMetricsQuery {
+        Socket *soc;
+        sMacAddr bssid;
+    } sApMetricsQuery;
+
+    typedef struct sApMetric {
+        sMacAddr bssid;
+        uint8_t channel_utilization;
+        uint16_t number_of_stas_currently_associated;
+        wfa_map::tlvApMetric::sEstimatedService estimated_service_parameters;
+        std::vector<uint8_t> estimated_service_info_field;
+    } sApMetric;
+    typedef struct sApMetricsResponse {
+        Socket *soc;
+        sMacAddr bssid;
+        sApMetric metric;
+    } sApMetricsResponse;
+
+    size_t processed_queries = 0;
+    std::vector<sApMetricsQuery> ap_metric_query;
```

Comment:
```suggestion
    std::vector<sApMetricsQuery> m_ap_metric_query;
```
(same for the others)